packages/utils/recursive-parser.test.js

```diff
@@ -25,3 +25,55 @@ defineTest(
 	},
 	"init"
 );
+
+// Tests for _add_ command
+defineTest(
+	__dirname,
+	"entry",
+	"fixture-2",
+	{
+		objects: "are",
+		super: [
+			"yeah",
+			{
+				test: new RegExp(/\.(js|vue)$/),
+				loader: "'eslint-loader'",
+				enforce: "'pre'",
+				include: ["customObj", "'Stringy'"],
+				options: {
+					formatter: "'someOption'"
+				}
+			}
+		],
+		nice: "':)'",
```

Comment:
Could you also adda a `Promise` to the test? Just to make sure that we cover every case
```js
{
  foo: Promise.resolve()
}
```

Reply:
Added! :+1: